src/form/Form.js

```diff
@@ -7,22 +7,22 @@ const StyledForm = styled.form`
 `;
 
 const StyledNameInput = styled.input`
-  padding: 10px 20px;
```

Comment:
You could style the input in a seperat file and then import the Input into your Form component, this would save you some styling lines